### DISCUSSION TITLE
Labels added for project maintenance

### DISCUSSION THREAD

**Discussion by cavasinf** [kevinpapst/TablerBundle#13] [General]
Hey @kevinpapst,
I'll now discuss from here [(Discussions)](https://github.com/kevinpapst/TablerBundle/discussions) to avoid polluting issues.

I've just added the new labels for the project.
It's based on what uses Symfony team, I think it works pretty well for my projects.

Tell me what you think about it.

**Comment by kevinpapst** [comment]
Looks good, thank you! I haven't setup much, because I did not expect any users at this point. And because I am really busy with  paid projects right now... so thank you for stepping in!
